apps/communications/contacts/js/bootstrap.js

```diff
@@ -33,6 +33,7 @@
        '/shared/js/contacts/import/utilities/config.js',
        '/contacts/js/utilities/extract_params.js',
        '/contacts/js/utilities/cookie.js',
+       '/contacts/js/utilities/object.js',
```

Comment:
We don't need to load this in the critical path, the function `isEmpty` is being used on the views:
detail
form
settings

So shouldn't need to be loaded on the bootstrap to save startup time.